build.sbt

```diff
@@ -371,6 +371,10 @@ lazy val unipublish =
     .aggregate(plugin) // Also publish the plugin when publishing this project
     .settings(name := (chisel / name).value)
     .enablePlugins(ScalaUnidocPlugin)
+    .settings(
+      // Plugin isn't part of Chisel's public API, exclude from ScalaDoc
+      ScalaUnidoc / unidoc / unidocProjectFilter := inAnyProject -- inProjects(plugin)
+    )
```

Comment:
I had to add this because the soft link of `SourceInfoFileResolver` was erroring during ScalaDoc generation saying the same class was included twice. I didn't realize Unidoc was even _trying_ to include the plugin since it's not part of this `unipublish` project (it's published separately). It never should've been included.